src/modules/recipes/RecipeList/index.tsx

```diff
@@ -0,0 +1,67 @@
+import {
+  useRecipes,
+  useRecipesDispatch,
+  RECIPE_ACTIONS,
+} from "../RecipesProvider";
+import { useEffect } from "react";
+import { Link } from "react-router-dom";
+import "./styles.css";
+
+// API'den dönen öğe tiplerini tanımlıyoruz
+interface Recipe {
+  idMeal: string;
+  strMeal: string;
+  strMealThumb: string;
+}
+
+export const RecipeList = () => {
+  const recipes = useRecipes();
+  const dispatch = useRecipesDispatch(); 
+
+  // useEffect ile sayfa yüklendiğinde API'den tarifleri çekecek
+  useEffect(() => {
+    const fetchRecipes = async () => {
+      try {
+        const response = await fetch(
+          ("https://www.themealdb.com/api/json/v1/1/filter.php?c=Seafood"),
```

Comment:
Use `fetchRecipesByCategory` from the api.ts file instead of doing a fetch request from scratch here